### ISSUE TITLE
Spyder crashes instantly, resetting & uninstalling doesn't help

### ISSUE THREAD

**Issue by stavroppdpls** [spyder-ide/spyder#3831]
No computer science or programming background, but I need Spyder to run some models for engineering classes, I apologize in advance for my complete lack of knowledge about these things.

Problem: Spyder crashes, --reset doesn't help, neither does uninstalling

How it happened:
1) I installed Anaconda 4.2.0 / Python 2.7, 64-bit. Windows 10
2) Spyder ran perfectly, but the model didn't recognize GAMS so I needed to add a path to import GAMS. 
3) I went to pythonpath manager, chose the path as I did in Matlab (C>GAMS>win64>24.7) and the manager asked me whether I want to synchronize. 
4) First I didn't sync because I wasn't sure what would change, (apologies I'm just really not good with these things). The model gave an import error, not recognizing the module GAMS
5) This time I synced the aforementioned path in the pythonpath manager and restarted Spyder. 
6) Since then, whatever I do, Spyder crashes and gives the "crashed during last session" message. 
7) I type "spyder --reset" in the command window, I see it removing files but it still doesn't work
8) I uninstalled Anaconda and deleted all traces I could find of Anaconda, Spyder, Python etc. (.ipython, .anaconda, anaconda2, .spyder, .matplotlib, .qtweb-something if I remember correctly) but it still doesn't work.

A text file of the spyder crash says the following:
"Traceback (most recent call last):
  File "C:\Users\micha\Anaconda2\lib\site-packages\spyder\app\mainwindow.py", line 2998, in main
    mainwindow = run_spyder(app, options, args)
  File "C:\Users\micha\Anaconda2\lib\site-packages\spyder\app\mainwindow.py", line 2912, in run_spyder
    main.post_visible_setup()
  File "C:\Users\micha\Anaconda2\lib\site-packages\spyder\app\mainwindow.py", line 1274, in post_visible_setup
    self.check_updates()
  File "C:\Users\micha\Anaconda2\lib\site-packages\spyder\app\mainwindow.py", line 2804, in check_updates
    from spyder.workers.updates import WorkerUpdates
  File "C:\Users\micha\Anaconda2\lib\site-packages\spyder\workers\updates.py", line 9, in <module>
    import ssl
  File "C:\Users\micha\Anaconda2\lib\ssl.py", line 125, in <module>
    from _ssl import HAS_SNI, HAS_ECDH, HAS_NPN, HAS_ALPN
ImportError: cannot import name HAS_ALPN
"

Googled for HAS_ALPN and couldn't find anything of use. 
So to recap: Spyder worked fine until I added something in the pythonpath manager and synchronized. resetting and uninstalling/reinstalling doesn't solve the problem. I can see the Spyder interface for about 1 or 2 seconds, after which it crashes.

Thank you in advance.
Stavro

**Comment by ccordoba12** [comment]
Did you also remove `C>GAMS>win64>24.7`?

**Comment by stavroppdpls** [comment]
I did not uninstall GAMS or remove anything GAMS-related. Would uninstalling and deleting all traces of both Anaconda and GAMS work?

**Comment by ccordoba12** [comment]
Yes, that could help. I mean, it's very strange that after removing Anaconda and all it's traces, Spyder is still failing to start.

**Comment by ccordoba12** [comment]
Closing because there's nothing we can about this in our side, sorry.
